api/src/services/extensions.ts

```diff
@@ -176,7 +176,9 @@ export class ExtensionsService {
 			return extension;
 		});
 
-		this.extensionsManager.reload();
+		this.extensionsManager.reload().then(() => {
+			this.extensionsManager.notifyOthers();
+		});
```

Comment:
This is getting a bit messy IMO 🙊 Can't we just move the publish call into the `reload` method?

Reply:
I was doing that so we had more control, but I understand your point 👍

Reply:
After some tests, if we put the publish call into `reload` method, all instances will be reloading forever. That's because one send to the others and the others send to others, so we cannot put that publish inside `reload`

Reply:
Well, we would have to break further reloads via an option in the https://github.com/directus/directus/blob/768e9088044ca7be0d79475bbfe695919fd1155a/api/src/extensions/manager.ts#L218 call, but the current solution with the more explicit naming works too 👍